docs/community.rst

```diff
@@ -0,0 +1,31 @@
+.. _community:
+
+*********
+Community
+*********
+
+We welcome your feedback, questions and ideas. Here's how to reach the community.
+
+Code of Conduct
+===============
+
+All communication and interactions in the Ansible Builder community are governed by the `Ansible code of conduct <https://docs.ansible.com/ansible/devel/community/code_of_conduct.html>`_. Please read and abide by it!
+Reach out to our community team at `codeofconduct@ansible.com <mailto:codeofconduct@ansible.com>`_ if you have any questions or need assistance.
+
+Ansible Forum
+=============
+
+Join the `Ansible Forum <https://forum.ansible.com>`_, the default communication platform for questions and help, development discussions, events, and much more. `Register <https://forum.ansible.com/signup?>`_ to join the community. Search by categories and tags to find interesting topics or start a new one; subscribe only to topics you need!
+
+* `Get Help <https://forum.ansible.com/c/help/6>`_: get help or help others. Please add appropriate tags if you start new discussions, for example `ansible-builder`, `ee`, and  `documentation`.
+* `Posts tagged with 'ansible-builder' <https://forum.ansible.com/tag/ansible-builder>`_: subscribe to participate in project-related conversations.
+* `Bullhorn newsletter <https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn>`_: used to announce releases and important changes.
+* `Social Spaces <https://forum.ansible.com/c/chat/4>`_: gather and interact with fellow enthusiasts.
+* `News & Announcements <https://forum.ansible.com/c/news/5>`_: track project-wide announcements including social events.
+
+For more information on the forum navigation, see `Navigating the Ansible forum <https://forum.ansible.com/t/navigating-the-ansible-forum-tags-categories-and-concepts/39>`_ post.
+
+Libera.Chat IRC
+===============
+
+Real-time conversations happen in the ``#ansible-builder`` Libera.Chat IRC channel.
```

Comment:
Is this true, though? Admittedly, I haven't used IRC since the bridge went away, but the Matrix channel for builder is totally silent. I feel like folks are going elsewhere (forums, perhaps).

Reply:
@Shrews let's remove it then

Reply:
@Shrews done, PTAL